src/sentry/api/bases/organization.py

```diff
@@ -430,7 +433,11 @@ def get_filter_params(
         try:
             if isinstance(project_ids, list):
                 project_ids = set(project_ids)
-            projects = self.get_projects(request, organization, project_ids=project_ids)
+            if isinstance(project_slugs, list):
+                project_slugs = set(project_slugs)
+            projects = self.get_projects(
+                request, organization, project_ids=project_ids, project_slugs=project_slugs
```

Comment:
How would you combine the two?
What would get ignored given a body param?

Looks like there's 3 ways to determine the list of projects to be fetched
- GET query params
- request body `project_ids`
- request body `project_slugs`

Each has their own valid use cases - so we _should_ continue to support each of them.

TBH the logic here looks like it could use a rewrite 🤔 
let me attempt to refactor this a bit.

Reply:
Yeah, it's pretty confusing. Thanks for trying.